web-client/src/ducks/myOffers/actions.ts

```diff
@@ -0,0 +1,77 @@
+import { IPost, Post } from 'src/models/posts';
```

Comment:
Created new directory src/ducks/myOffers, follows same pattern as myRequests.  See comments above for myRequests.